parser-typechecker/src/Unison/Codebase/SqliteCodebase/Migrations.hs

```diff
@@ -68,12 +72,19 @@ ensureCodebaseIsUpToDate localOrRemote root getDeclType termBuffer declBuffer co
           for_ (Map.toAscList migrationsToRun) $ \(SchemaVersion v, migration) -> do
             putStrLn $ "🔨 Migrating codebase to version " <> show v <> "..."
             run migration
-          pure (not (null migrationsToRun))
+          let ranMigrations = not (null migrationsToRun)
+          when ranMigrations $ do
+            putStrLn $ "🕵️  Checking codebase integrity..."
+            run do
+              integrityCheckFullCodebase >>= \case
```

Comment:
Here's where we now run a full integrity check after any migrations.

It's pretty fast; only seems to take a few seconds.